kong-1.1.2-0.rockspec

```diff
@@ -43,6 +43,7 @@ dependencies = {
   "kong-plugin-serverless-functions ~> 0.2",
   "kong-prometheus-plugin ~> 0.3",
   "kong-proxy-cache-plugin ~> 1.2",
+  "kong-plugin-session == 1.0.2-3",
```

Comment:
We don't pin these plugins versions since we control their release cycle, and leaving the patch version unspecified allows us to release bugfixes to those installation methods that rely on `luarocks install` without having to cut a whole new Kong release (we also never include the revision number since it should be left to the maintainer's discretion to update if necessary). Could we specify `~> 1.0` here instead?